packages/react-lab/src/date-picker/Calendar/Calendar.js

```diff
@@ -0,0 +1,104 @@
+import { Box } from '@tonic-ui/react';
+import { usePrevious } from '@tonic-ui/react-hooks';
+import isSameMonth from 'date-fns/isSameMonth';
+import isSameYear from 'date-fns/isSameYear';
+import React, { forwardRef, useEffect, useState } from 'react';
+import DecadeView from './DecadeView';
+import MonthView from './MonthView';
+import YearView from './YearView';
+import Navigation from './Navigation';
+import { useCalendarStyle } from './styles';
+import {
+  isDate,
+  toDate,
+} from '../utils';
+
+const Calendar = forwardRef((
+  {
+    calendarStartDay = 0, // 0 = Sunday, 1 = Monday, ...
+    children,
+    dateFormat = 'yyyy-MM-dd',
+    value = new Date(),
+    view = 'month', // one of 'month', 'year', 'decade'
```

Comment:
You don't need to provide the `view` prop for this component. This is a kind of initial value rather than a controlled state, or you should rename it to `defaultView`.